packages/core/src/components/Table/Body/Body.tsx

```diff
@@ -34,13 +42,18 @@ const Body: FC<TableBodyProps> = memo(props => {
         isCollapseKeyPressed = useKeyPress(keyBindings.collapseRow!, false, tableRef);
 
     useEffect(() => {
-        data.length && isUpKeyPressed && setCursor(prevState => (prevState > 0 ? prevState - 1 : prevState));
+        !customCursor && data.length && isUpKeyPressed && setCursor(prevState => (prevState > 0 ? prevState - 1 : prevState));
```

Comment:
You need to add this in useEffect's dependency also

Reply:
Resolved